blog.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="styles/stylesheet.css">
+    <title>Blog</title>
+  </head>
+
+  <body>
+    <header class="header">
+      <div class="checkout">
+      <a href="checkout.html"><img src="images/cart-icon.svg" alt="cart icon"></a>
+      <p>$1234.56</p>
+      <input type="button" value="Checkout">
```

Comment:
Non of your checkout buttons are linked to your checkout page. There is no way to visit unless you know the address.